### DISCUSSION TITLE
Using external storage as primary with AIO

### DISCUSSION THREAD

**Discussion by Sid127** [nextcloud/all-in-one#1330] [Questions] [answered]
I have a nextcloud AIO instance set up on a cheap small VM that doesn't have nearly enough storage, which I'm using with Wasabi to fulfill my requirement. Would it be possible to have my external storage be the primary storage instead? I know it is possible by editing the config.php on a nextcloud server, but I'm not sure about how AIO works

**Comment by szaimen** [comment, marked as answer]
Hi, I think this should be possible by choosing the manual install: https://github.com/nextcloud/all-in-one/tree/main/manual-install

We basically offer the same variables for our Nextcloud container like the docker container: https://github.com/nextcloud/docker#:~:text=to%20use%20an%20external%20s3%20compatible%20object%20store%20as%20primary%20storage%2C%20set%20the%20following%20variables

![image](https://user-images.githubusercontent.com/42591237/198386732-7742d55c-5364-43a9-b4b7-32e4042f34bb.png)
